src/routes/accounts.ts

```diff
@@ -56,11 +56,8 @@ export function accountsRouter({
       ) => {
         // Delegate to type-specific handlers after validation provides type guards
         switch (req.params.fiatAccountSchema) {
-          case FiatAccountSchema.MockCheckingAccount:
-            validateSchema<MockCheckingAccount>(
-              req.body,
-              'MockCheckingAccountSchema',
-            )
+          case FiatAccountSchema.AccountNumber:
+            validateSchema<AccountNumber>(req.body, 'AccountNumberSchema')
```

Comment:
Should there be a `break` after this?

Reply:
yeah.. was going to put that in another PR but since this line is changed anyway I might as well